docs/software/markdown-migration.md

```diff
@@ -0,0 +1,150 @@
+
+#### Highlighting user input  ####
+
+Within command blocks and file snippets, we've used `%RED%...%ENDCOLOR%`, `&lt;...&gt;`, etc. to highlight areas that users would have to insert text specific to their site. For now, use desciptive, all-caps text wrapped in angle brackets to indicate user input:
+
+```
+<USER INPUT>
+```
+
+#### Ordered Lists ####
+
+Ordered lists are often broken up into multiple lists if there are command blocks/file snippets and/or additional text within one of the list items. To make sure the contents of an item are indented properly, use the following formatting:
+
+- For code blocks or file snippets, add an empty line after any regular text, then insert `(N+1)*4` spaces at the beginning of each line, where N is the level of the item in the list.
+- For additional text (i.e. after a code block), insert `N*4` spaces at the beginning of each line, where N is the level of the item in the list.
+
+For example:
+
+```
+1. Foo
+    - Bar
+
+            COMMAND
+            BLOCK
+        text associated with Bar
+
+    text associated with Foo
+
+2. Baz
+
+        FILE
+        SNIPPET
+
+```
+
+There are 12 spaces and 8 spaces in front of the command block and text associated with `Bar`, respectively; 4 spaces in front of the text associated with `Foo`; and 8 spaces in front of the file snippet associated with `Baz`
+
+### Notes ###
+
+To catch the user's attention for important items or pitfalls, we used `%NOTE%` TWiki macros, these can be replaced with admonition-style notes:
+
+```
+!!! note
+    # things to note
+```
+
+### Obvious errors ###
+
+If you see any other obvious errors (e.g., links to gratia web), feel free to correct them while you're editing the doc *iff* the changes take less than ~15 minutes. This isn't a renovation project!
+
+Making the pull request
+-----------------------
+
+1. Create a branch based off of master
+2. Add page to [mkdocs.yml](../../mkdocs.yml) 
+3. Commit your changes and push to your GH repo
+4. Make PR containing the following tasks in the body body:
+
+        - [ ] Remaining TWiki link #1
+        - ...
+        - [ ] Remainng TWiki link #N
+        - [ ] Add migration header to TWiki document
+
+Adding a header to the TWiki document
+-------------------------------------
+
+After completing the migration of a document, replace the contents of TWiki document with the following header, linking to the location of the migrated document:
+
+```
+<div style="border: 1px solid black; margin: 1em 0; padding: 1em; background-color: #FFDDDD; font-weight: 600;">
+This is an archive, find the new version of this document [[LINK TO GITHUB DOCUMENT][here]].
+
+Background:
+
+At the end of year (2017), the TWiki will be retired in favor of !GitHub. You can find the various TWiki webs and their new !GitHub locations listed below:
+
+   * Release3: https://opensciencegrid.github.io/docs
+   * !SoftwareTeam: https://opensciencegrid.github.io/technology
+</div>
+```
```

Comment:
If we're replacing the entirety of the TWiki document's contents, what links would there be to update?